test/testcafe/spec/IdentifyWithThirdPartyIdps_spec.js

```diff
@@ -143,7 +143,7 @@ test.requestHooks(logger, mockOnlyOneIdp)('should auto redirect to 3rd party IdP
 test.requestHooks(logger, mockOnlyOneIdp)('Direct auth: does not auto redirect to 3rd party IDP on initial load', async t => {
   const identityPage = await setupDirectAuth(t);
 
-  // await t.wait(100);
+  await t.wait(100);
```

Comment:
I noticed a lot of Bacon flakiness with this test, and uncommenting this seems to make it stable.

Reply:
Yeah I did try to debug this quite a bit locally and the first thing I tried was to wait for the form render with the social auth buttons but that didn't seem to help. Tbh, I wasn't quite sure why that didn't work and a specific wait for 100ms did. But I can try to investigate some more.